webapp/containers/App/App.js

```diff
@@ -107,48 +110,35 @@ class App extends PureComponent {
   }
 
   render() {
-    const {
-      sidebarNavItems,
-      location,
-      theme,
-      match,
-      currentClient
-    } = this.props;
+    const { sidebarNavItems, location, theme, match } = this.props;
     return (
       <ThemeProvider theme={theme}>
-        {currentClient.id ? (
-          <div>
-            <div
-              className={`${theme.app.container} container-fluid`}
-              role="main"
-            >
-              <div className="row">
-                <div
-                  className={`${theme.app.sidebarContainer} col-sm-4 col-lg-3`}
-                >
-                  <Sidebar
-                    sidebarNavItems={sidebarNavItems}
-                    location={location}
-                    theme={theme}
-                    match={match}
-                  />
-                </div>
-                <div className={`${theme.app.content} col-sm-8 col-lg-9`}>
-                  <Header />
-                  <Route
-                    exact
-                    path="/"
-                    render={() => <Redirect to={`/${this.getHomePath()}`} />}
-                  />
-                  <Panel />
-                </div>
+        <div>
```

Comment:
Can you turn this into a `<React.Fragment>`?
https://reactjs.org/docs/fragments.html